sql-plugin/src/main/java/com/nvidia/spark/rapids/RapidsHostColumnVectorCore.java

```diff
@@ -22,6 +22,7 @@
 
 import org.apache.spark.sql.types.ArrayType;
 import org.apache.spark.sql.types.DataType;
+import org.apache.spark.sql.types.DataTypes;
```

Comment:
Are they related?

Reply:
`org.apache.spark.sql.types.DataType` refers to the abstract Scala class: [DataType](https://github.com/apache/spark/blob/HEAD/sql/catalyst/src/main/scala/org/apache/spark/sql/types/DataType.scala), who is the parent of all specific data types.
`org.apache.spark.sql.types.DataTypes` refers to a Java class: [DataTypes](https://github.com/apache/spark/blob/HEAD/sql/catalyst/src/main/java/org/apache/spark/sql/types/DataTypes.java), who holds the singletons representing all data types.